src/legacy/core_plugins/kibana/migrations/migrations.test.js

```diff
@@ -1180,6 +1180,121 @@ Array [
       expect(migratedDoc).toEqual({ attributes: { visState: JSON.stringify(expected) } });
     });
   });
+  describe('7.5.0 tsvb split_filters migration', () => {
+    const migrate = doc => migrations.visualization['7.5.0'](doc);
+    const generateDoc = ({ params }) => ({
+      attributes: {
+        title: 'My Vis',
+        description: 'This is my super cool vis.',
+        visState: JSON.stringify({ params }),
+        uiStateJSON: '{}',
+        version: 1,
+        kibanaSavedObjectMeta: {
+          searchSourceJSON: '{}',
+        },
+      },
+    });
+    it('should change series item filters from a string into an object for all filters', () => {
+      const params = {
+        type: 'timeseries',
+        filter: {
+          query: 'bytes:>1000',
+          language: 'lucene'
+        },
+        series: [
+          {
+            split_filters: [{ filter: 'bytes:>1000' }],
+          },
+        ]
+      };
+      const timeSeriesDoc = generateDoc({ params: params });
+      const migratedtimeSeriesDoc = migrate(timeSeriesDoc);
+      const timeSeriesParams = JSON.parse(migratedtimeSeriesDoc.attributes.visState).params;
+      expect(Object.keys(timeSeriesParams.filter)).toEqual(
+        expect.arrayContaining(['query', 'language'])
+      );
+      expect(Object.keys(timeSeriesParams.series[0].split_filters[0].filter)).toEqual(
+        expect.arrayContaining(['query', 'language'])
+      );
+    });
+    it('should change series item split filters when there is no filter item', () => {
+      const params = {
+        type: 'timeseries',
+        filter: {
+          query: 'bytes:>1000',
+          language: 'lucene'
+        },
+        series: [
+          {
+            split_filters: [{ filter: 'bytes:>1000' }],
+          },
+        ],
+        annotations: [
+          {
+            query_string: {
+              query: 'bytes:>1000',
+              language: 'lucene'
+            }
+          }
+        ],
+      };
+      const timeSeriesDoc = generateDoc({ params: params });
+      const migratedtimeSeriesDoc = migrate(timeSeriesDoc);
+      const timeSeriesParams = JSON.parse(migratedtimeSeriesDoc.attributes.visState).params;
+      expect(timeSeriesParams.series[0].split_filters[0].filter).toHaveProperty('query');
+    });
+    it('should not error out if there are no split filter items', () => {
+      const params = {
+        type: 'timeseries',
+        filter: {
+          query: 'bytes:>1000',
+          language: 'lucene'
+        },
+        series: [
+          {
+            split_filters: [],
+          },
+        ]
+      };
+      const timeSeriesDoc = generateDoc({ params: params });
+      const migratedtimeSeriesDoc = migrate(timeSeriesDoc);
+      const timeSeriesParams = JSON.parse(migratedtimeSeriesDoc.attributes.visState).params;
+      expect(timeSeriesParams.series[0].split_filters).not.toHaveProperty;
```

Comment:
We're not calling the `toHaveProperty` function here actually, so the test case doesn't contain any valid `expect`.

Reply:
Fixed: https://github.com/elastic/kibana/pull/49000/files#diff-5afd24840636884e72ed2920f5e67a71R1264